src/ucp/api/ucp_def.h

```diff
@@ -385,13 +385,30 @@ typedef void (*ucp_tag_recv_callback_t)(void *request, ucs_status_t status,
  * @brief UCP worker wakeup events mask.
  *
  * The enumeration allows specifying which events are expected on wakeup, though
- * empty events are possible.
+ * empty events are possible for any type of events excepting @ref UCP_WAKEUP_TX
+ * and UCP_WAKEUP_RX which are the most common events.
+ *
+ * @note UCP doesn't generate POLLOUT-like events (see poll man pages) since
+ * it's always ready to initiate outgoing operations (e.g any type of sending,
+ * atomic or RMA). Hoverer, they can be queued due to limitation of resources.
+ * Send completions are reported by POLLIN-like events.
```

Comment:
Suggested re-wording:

`@note` Send completions are reported by POLLIN-like events (see poll man
page). Since outgoing operations can be initiated at any time, UCP
does not generate POLLOUT-like events, although it must be noted that
outgoing operations may be queued depending upon resource availability.